spec/fixtures/credit_line_closing_info.ofx

```diff
@@ -0,0 +1,403 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<?OFX OFXHEADER="100" VERSION="102" SECURITY="NONE" OLDFILEUID="NONE" NEWFILEUID="NONE"?>
+<OFX>
+ <SIGNONMSGSRQV1>
+  <SONRQ>
+   <DTCLIENT>20160212234833.123[-5:EST]</DTCLIENT>
+   <ACCESSTOKEN>774bcce2-076a-4ba7-8cdd-9692d05a1645</ACCESSTOKEN>
+   <LANGUAGE>ENG</LANGUAGE>
+   <APPID>INTU</APPID>
+   <APPVER>0100</APPVER>
+   <APPKEY>JPMINTMIN100001</APPKEY>
+  </SONRQ>
+ </SIGNONMSGSRQV1>
+ <STMTTRNRQ>
+  <TRNUID>70f384d0-daaf-4f92-838f-04f441219cb8</TRNUID>
+  <STMTRQ>
+   <BANKACCTFROM>
+    <BANKID>111777882</BANKID>
+    <ACCTID>4307754</ACCTID>
+    <ACCTTYPE>CREDITLINE</ACCTTYPE>
+   </BANKACCTFROM>
+   <INCTRAN>
+    <DTSTART>20091129</DTSTART>
+    <DTEND>20180327</DTEND>
+    <INCLUDE>Y</INCLUDE>
+   </INCTRAN>
+   <INCLUDEPENDING>Y</INCLUDEPENDING>
+  </STMTRQ>
+ </STMTTRNRQ>
+ <STMTENDTRNRQ>
+  <TRNUID>edbe7630-f366-4d80-868c-eee16c5eb202</TRNUID>
+  <STMTENDRQ>
+   <BANKACCTFROM>
+    <BANKID>111777882</BANKID>
+    <ACCTID>4307754</ACCTID>
+    <ACCTTYPE>CREDITLINE</ACCTTYPE>
+   </BANKACCTFROM>
+  </STMTENDRQ>
+ </STMTENDTRNRQ>
+</OFX>
+
+        Response:-
```

Comment:
Does this file contains 2 xml files?

Reply:
It had both request and response, removed request xml, as it is not needed